modules/eco-poison-pill-operator-about.adoc

```diff
@@ -32,7 +33,23 @@ metadata:
 spec:
   safeTimeToAssumeNodeRebootedSeconds: 180 <1>
   watchdogFilePath: /test/watchdog1 <2>
+  isSoftwareRebootEnabled: true <3>
+  apiServerTimeout: 15s <4>
+  apiCheckInterval: 5s <5>  
+  maxApiErrorThreshold: 3 <6>
+  peerApiServerTimeout: 5s <7>
+  peerDialTimeout: 5s <8>
+  peerRequestTimeout: 5s <9>
+  peerUpdateInterval: 15m <10>
 ----
 
 <1> Specify the timeout duration for the surviving peer, after which the Operator can assume that an unhealthy node has been rebooted. The Operator automatically calculates the lower limit for this value. However, if different nodes have different watchdog timeouts, you must change this value to a higher value.
-<2> Specify the file path of the watchdog device in the nodes. If a watchdog device is unavailable, the `PoisonPillConfig` CR uses a software reboot.
\ No newline at end of file
+<2> Specify the file path of the watchdog device in the nodes. If a watchdog device is unavailable, the `PoisonPillConfig` CR uses a software reboot.
```

Comment:
again - is this a specific feature? if it's not, it might be more straightforward to say "node reboot".